### PR TITLE
Fixes external logging data

### PR DESCRIPTION
**Pull request by jungeunyoon**
It's best to Stringify here so that we can use the actual object for data verification and testing in the other modules and this module before we log to BQ.

### CODE REVIEW

**Comment by jungeunyoon** [comment]
@Rise-Vision/delivery  Actually guys, one sec. This wasn't ready for review. Let me manually test.

File: src/logger.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ module.exports = {
 
     entry.data.ts = nowDate.toISOString();
 
-    return bqControllers[key].log(entry.table, entry.data, nowDate, entry.suffix)
+    return bqControllers[key].log(entry.table, JSON.stringify(entry.data), nowDate, entry.suffix)
```
Comment (santiagonoguez) [inline src/logger.js]: isn't that bqController a function like this one ?
https://github.com/Rise-Vision/common-display-module/blob/master/external-logger.js#L46
in that case it is expecting an object, not a string

**Comment by stulees** [comment]
@jungeunyoon This change will break the insertion code for the actual insert to BQ, see here - https://github.com/Rise-Vision/logging-module/blob/master/src/bq/bq-client.js#L33-L57 . I believe the change to fix .log files common-electron `logger` (or potentially common-display-module external-logger) not in Logging module. @santiagonoguez FYI

**Comment by jungeunyoon** [comment]
@stulees @santiagonoguez Yeah I misread the issue tracking doc and realized it pretty quickly.

**Pull request closed without merging**